react/src/components/blocks/ChatWindow/MessageForm/SubmitButton/SubmitButton.js

```diff
@@ -1,10 +1,27 @@
 import React from 'react'
 import styles from './SubmitButton.styles.css'
 
+import sendImg from '../../../../../static/images/send.svg'
+import plusImg from '../../../../../static/images/plus.svg'
+import videoImg from '../../../../../static/images/video.svg'
+import micImg from '../../../../../static/images/mic.svg'
+import smileyImg from '../../../../../static/images/smiley.svg'
+import atImg from '../../../../../static/images/at.svg'
+import fontImg from '../../../../../static/images/aafont.svg'
+
+
 function SubmitButton() {
   return (
     <>
-      <p className={styles.line}>SubmitButton goes here.</p>
+      <div class={styles.footerToolBar}>
+      <img src={plusImg} alt="send" className={styles.footerToolbarImg} />
+      <img src={videoImg} alt="video" className={styles.footerToolbarImg} />
+      <img src={micImg} alt="mic" className={styles.footerToolbarImg}/>
+      <img src={smileyImg} alt="smiley" className={styles.footerToolbarImg}/>
+      <img src={atImg} alt="at" className={styles.footerToolbarImg} />
+      <img src={fontImg} alt="font" className={styles.footerToolbarImg}/>
+      <img src={sendImg} alt="send"  className={styles.sendImg}/>
+      </div>
```

Comment:
can this indented properly. plz run prettier. get the alternative text from i18n. 

```
alt="smiley"
```

how about keeping all the images in array and map them like below.

### In Util File.
```
import sendImg from '../../../../../static/images/send.svg'

const getToolBarButtonImages = [
  {
     sourceImage: sendImg,
     alternativeTextLocaleKey: 'localeKey',
     styledClassName: 'sendImg'
  },
 ...
]
```

### In Component.

```
import { getToolBarButtonImages } from './path/to/util'

return (
  <>
    <div class={styles.footerToolBar}>
      {getToolBarButtonImages.map(({ sourceImage, alternativeTextLocaleKey, styledClassName }) => {
        return <img src={image.src} alt={formatMessage({ id: alternativeTextLocaleKey })} className={styles[className]} />
      })}
    </div>
  </>
)
```

Reply:
The requested change is addressed.